apps/rush-lib/src/data/CommandLineConfiguration.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
>// [](start = 0, length = 2)

Where did this file come from?  What does it have to do with strict null checks?

Reply:
It was accidently added, is a part of the next PR I was working on

---
In reply to: [142773840](https://github.com/Microsoft/web-build-tools/pull/385#discussion_r142773840) [](ancestors = 142773840)